src/modules/aws_ecr/index.ts

```diff
@@ -626,11 +622,11 @@ class RepositoryPolicyMapper extends MapperBase<RepositoryPolicy> {
     update: (es: RepositoryPolicy[], ctx: Context) => ctx.orm.save(RepositoryPolicy, es),
     delete: (es: RepositoryPolicy[], ctx: Context) => ctx.orm.remove(RepositoryPolicy, es),
     read: async (ctx: Context, id?: string) => {
-      const [repositoryName, region] = id?.split('|') ?? [undefined, undefined];
-      const opts = repositoryName
+      const [repositoryId, region] = id?.split('|') ?? [undefined, undefined];
+      const opts = repositoryId
         ? {
             where: {
-              repositoryName,
+              repositoryId,
```

Comment:
If I'm not wrong this won't work, because the `RepositoryPolicy` entity does not have a `repositoryId` property. I think it would be something like this:
```suggestion
              repository: { id: repositoryId },
```